qa/rpc-tests/wallet_addresses.py

```diff
@@ -39,39 +39,39 @@ def addr_checks(default_type):
         # Current height = 200 -> Sprout
         # Default address type is Sprout
```

Comment:
Comment is now out-of-date.

Reply:
Added this to https://github.com/zcash/zcash/issues/3686

Reply:
Scratch that, I will fix now.